docs/guides/main.rst

```diff
@@ -142,7 +142,8 @@ place to start::
 
 Change that first path to a directory where you'd like to keep your music. Then,
 for ``library``, choose a good place to keep a database file that keeps an index
-of your music. (The config's format is `YAML`_. You'll want to configure your
+of your music. Beets will prompt you if the parental directories for database do
+not exist. (The config's format is `YAML`_. You'll want to configure your
```

Comment:
TBH I think it's probably OK to leave this sentence off? This is an edge case, and the prompt clearly explains to users what's going on, so I don't think we need to warn people that this might happen.